packages/pwa-kit-create-app/assets/bootstrap/js/overrides/app/constants.js.hbs

```diff
@@ -0,0 +1,18 @@
+// it's necessary that the interface remain consistent, as other
+// files in the base template rely on constants.js, thus we
+// import the underlying constants.js and re-export it
```

Comment:
```suggestion
/* 
    Hello there! This is a demonstration of how to override a file from the base template.
    
    It's necessary that the module export interface remain consistent, 
    as other files in the base template rely on constants.js, thus we
    import the underlying constants.js, modifies it and re-export it.
*/
```

Reply:
It's really good to have more comments in the generated projects to guide new users.